plugin/evm/vm.go

```diff
@@ -484,6 +476,43 @@ func (vm *VM) Initialize(
 		}
 	}
 
+	// Define the default set of validators.
+	validators := []ids.ShortID{}
+
+	// Initialize the FTSO system, which is responsible for all of our interactions
+	// with the FTSO smart contracts running at the EVM level.
+	blockchain := vm.chain.BlockChain()
+	ftso, err := NewFTSOSystem(blockchain,
+		common.HexToAddress("0x1000000000000000000000000000000000000003"),
```

Comment:
I think we should declare these as `const`

Reply:
This is done in the other PR.

Reply:
This is done now.